inject/apple.js

```diff
@@ -18,7 +19,15 @@ function initCode () {
         trackerLookup: processedConfig.trackerLookup,
         documentOriginIsTracker: isTrackerOrigin(processedConfig.trackerLookup),
         site: processedConfig.site,
-        bundledConfig: processedConfig.bundledConfig
+        bundledConfig: processedConfig.bundledConfig,
+        constructMessagingConfig: (context) => {
+            if (import.meta.injectName !== 'apple-isolated') throw new Error('Only apple-isolated messaging supported')
+            return new WebkitMessagingConfig({
+                webkitMessageHandlerNames: [context.context],
```

Comment:
in C-S-S, this is a known value, so I'd remove the wiring of context and just hard-code it here, or get it from `import.meta` or something. I only re-used the context name because I could (at the time) but if it's awkward let's remove.

Reply:
We actually don't need this to be special anywhere 'debug' isn't used by the extension so we can simplify the context creation too.

Reply:
Removed it being a method entirely due to the removal of the context too.